module/omp_lib.h

```diff
@@ -0,0 +1,548 @@
+! Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
1) Can we use the headers provided by the OpenMP library?
2) Can this Copyright be removed? I am guessing that for the fixed-form case this header will be directly placed included the source. Alternatively, can this be preprocessed and included?
3)Which OpenMP version does this header correspond to?